client/my-sites/post-relative-time-status/style.scss

```diff
@@ -1,18 +1,18 @@
 .post-relative-time-status {
-	.time,
-	.status {
+	.post-relative-time-status__time,
+	.post-relative-time-status__status {
 		display: inline-block;
 		margin-right: (11 / 12) * 1em;
```

Comment:
>Do we still need this rule?

I believe this may have been for the drafts list, where actions are shown under an ellipsis menu adjacent the relative time status on touch devices. That said, there are two issues: (a) if that's the desired layout, the rule should be specific to the drafts list and not this component and (b) it appears to be broken in master currently anyways (non-production feature, so not urgent, but cc @mtias).

![image](https://cloud.githubusercontent.com/assets/1779930/16897839/f6b0e878-4b73-11e6-97c8-57ab36d159dc.png)

For that reason, will go ahead and remove this.